### PR TITLE
General fixes

### PR DESCRIPTION
**Pull request by Elfoniok**

### CODE REVIEW

File: apps/core/task/coretask.py [hunk 1]
```diff
@@ -145,7 +145,7 @@ def __init__(self,
             subtask_timeout=task_definition.subtask_timeout,
             resource_size=self.resource_size,
             estimated_memory=task_definition.estimated_memory,
-            max_price=task_definition.max_price,
+            max_price=task_definition.max_price
```
Comment (jiivan) [inline apps/core/task/coretask.py]: :-1:

File: golem/environments/environmentsmanager.py [hunk 2]
```diff
@@ -6,7 +6,8 @@
 
 
 class EnvironmentsManager(object):
-    """ Manage known environments. Allow user to choose accepted environment, keep track of supported environments """
+    """ Manage known environments. Allow user to choose accepted environment,
+    keep track of supported environments """
```
Comment (jiivan) [inline golem/environments/environmentsmanager.py]: First line should be a summary without a continuation

**Review by jiivan — APPROVED** [review]

File: apps/core/task/coretask.py (hunk 1 above)
Comment (Elfoniok) [reply apps/core/task/coretask.py]: Ok

File: golem/environments/environmentsmanager.py (hunk 2 above)
Comment (Elfoniok) [reply golem/environments/environmentsmanager.py]: Ok

**Pull request merged**